lib/release_notes/changelog_file.rb

```diff
@@ -35,8 +35,8 @@ def metadata
 
     private
 
-    def github_file
-      @api.find_content(@file_path)
+    def github_file(repo = nil)
```

Comment:
Note to self: This is further proof that maybe I should get rid of this Changelog file and all of this should be in the manager.